packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.test.ts

```diff
@@ -174,7 +173,8 @@ describe("inviteMemberHandler", () => {
           autoAcceptEmailDomain: null,
         });
 
-        expect(paymentsMock.updateQuantitySubscriptionFromStripe).toHaveBeenCalledWith(input.teamId);
+        // TODO: Fix this test
```

Comment:
This test isnt broke it just is being called in the teamRepo.updateQuanitty test now. 

Should probably just remove it from this test file.

Reply:
Agreed